RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelClusterThresholds.h

```diff
@@ -9,6 +9,6 @@ struct SiPixelClusterThresholds {
   const int32_t otherLayers;
 };
 
-constexpr SiPixelClusterThresholds kSiPixelClusterThresholdsDefaultPhase1{.layer1 = 2000, .otherLayers = 4000};
+constexpr SiPixelClusterThresholds kSiPixelClusterThresholdsDefaultPhase1{.layer1 = 4000, .otherLayers = 4000};
```

Comment:
we need a modifier for Run2 then

Reply:
It needs to be configured @dkotlins not change the default

Reply:
what about older Run3 Relvals?

Reply:
Too bad